.gitignore

```diff
@@ -1 +1,2 @@
 .idea/
+newline
```

Comment:
Please fix this to a proper phyton ignore file. 
Use this: https://github.com/github/gitignore/blob/master/Python.gitignore